release_notes.md

```diff
@@ -0,0 +1,67 @@
+ # Release notes
+
+- Woof Mobile APP
+- Release notes version: 1
+- Release note date: May. 4, 2023
+
+All notable changes to this application will be documented in this file.
+
+## Overview
+
+This is the application for iOS.
+Woof application connects dog owners with pet sitters which contributes to build trusted relationships between them. The app makes it easy for nearby pet owners to find pet sitting and makes convienience of responding to jobs for pet sitters from phone.
+App also makes managing sitting bussines and pet owner's schedule. They can choose their own schedule, prices, and requirements for servise. 
+
+## [Unreleased]
+
+`More information about working with the application will be added during development.`
+
+### User impacts
+
+Software requirements: 
+
+ * Deployment target for iOS SDKs - 15.0
+ * Supported devices - iPhone only
+ * Supported modes - portrait
+ * Dark/night theme - deactivate
+ * The linters (Swiftformat and Swiftlint) integrated with Xcode in the build phase.
+
+### Notes
+
+Anyone can download and use the app features.
+
+### Support
+
+You can contact the authors by the following GitHub's usernames:
+
+- Nadia Shpakouskaya: [@NadzeyaShpakouskaya](https://github.com/NadzeyaShpakouskaya)
+- Alex Zavadskiy: [@iBYarchitect](https://github.com/iBYarchitect)
+- Julya Kmita: [@julyakmita](https://github.com/julyakmita)
+
+### Added 
+- Users can now download the initial Xcode project.
+
+## [0.1.0] - 2023-03-05
+
+`Information about working with the application will be added during development.`
+`Possible sections to fill are presented below`
+
+### Changes
+* 
+* 
+
+### Features
+* 
+* 
+
+### Bug Fixes
+* 
+* 
+
+### Performance Improvements
+* 
+* 
+
+### Other Changes
+* 
+* 
```

Comment:
Why do you need these empty sections here?

Reply:
I thought, it would help for filling in the future. But you are right. Now it's irrelevant. The empty sections are deleted.